<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBarSetDlg</class>
 <widget class="QDialog" name="ToolBarSetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>相关设置：</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="2,3" columnminimumwidth="3,2">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>7</number>
     </property>
     <property name="horizontalSpacing">
      <number>7</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>色值RGB565转RGB888:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>toolbar高度：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>subtoolbar高度：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>subtoolbar宽度：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBox_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>toolbar宽度：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBox_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>文件索引.txt改成.ui：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="cbxRGBChange">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>关闭</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>开启</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="cbxTxtChange">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>关闭</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>开启</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>图片关联资源qrc文件：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBox</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>cbxRGBChange</tabstop>
  <tabstop>cbxTxtChange</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancle</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
